javascript/builtins/String.json

```diff
@@ -1299,45 +1299,98 @@
             "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String/match",
             "spec_url": "https://tc39.es/ecma262/#sec-string.prototype.match",
             "support": {
-              "chrome": {
-                "version_added": "1"
-              },
-              "chrome_android": {
-                "version_added": "18"
-              },
+              "chrome": [
+                {
+                  "version_added": "64"
+                },
+                {
+                  "version_added": "1",
+                  "version_removed": "64",
+                  "notes": "No support for named capturing groups."
+                }
+              ],
+              "chrome_android": [
+                {
+                  "version_added": "64"
+                },
+                {
+                  "version_added": "18",
+                  "version_removed": "64",
+                  "notes": "No support for named capturing groups."
+                }
+              ],
               "edge": {
-                "version_added": "12"
+                "version_added": "12",
+                "notes": "No support for named capturing groups."
               },
               "firefox": {
-                "version_added": "1"
+                "version_added": "1",
+                "notes": "No support for named capturing groups."
               },
               "firefox_android": {
-                "version_added": "4"
+                "version_added": "4",
+                "notes": "No support for named capturing groups."
               },
               "ie": {
-                "version_added": "4"
-              },
-              "nodejs": {
-                "version_added": true
-              },
-              "opera": {
-                "version_added": true
-              },
-              "opera_android": {
-                "version_added": true
+                "version_added": "4",
+                "notes": "No support for named capturing groups."
               },
+              "nodejs": [
+                {
+                  "version_added": "10.0.0"
+                },
+                {
+                  "version_added": "1",
+                  "version_removed": "10.0.0",
+                  "notes": "No support for named capturing groups."
+                }
+              ],
+              "opera": [
+                {
+                  "version_added": "51"
+                },
+                {
+                  "version_added": "1",
+                  "version_removed": "51",
+                  "notes": "No support for named capturing groups."
+                }
+              ],
+              "opera_android": [
+                {
+                  "version_added": "47"
+                },
+                {
+                  "version_added": "1",
```

Comment:
Should be “14” since that’s when it switched to Blink.

Reply:
Surely Presto+Futhark/Carakan et al would've supported `string.prototype.match` though? It doesn't seem like something that only V8/Blink would've supported. Or do we not care so much about the older engines?

Reply:
Presto's so old that we haven't really cared much for its data here in BCD, but we've decided not to remove it since we might as well keep the data around if we have it (and others can submit PRs if they have the actual version numbers).  However, we cannot assume that support was introduced in the very first Opera versions, and since we're mirroring data from Chrome, we set it to the first Blink-powered version.

(Note: I have a PR open to add ranged versions for Opera, so that instead of just "14" or "15", it'll be "≤14" or "15", meaning "we know this feature was at least supported since then but it could have been implemented earlier; these are versions so old that hardly anyone will actually care about this data".  I do also plan on spending some time going through the old Opera versions, because why not, right?  😛)

Reply:
@vinyldarkscratch doesn't this suggest that interim solution is `true` rather than "1" _or_ "14" (until we have the version ranges)? If we don't know whether it shipped with Presto or Blink, then we don't really have a version number for this.

Reply:
I guess you have a point there, yeah, although so far we've been setting it to "14/15" throughout BCD.  `true` is probably better though!